webapp/Pages/ContactUs.razor

```diff
@@ -0,0 +1,91 @@
+@page "/contactus"
+
+<section id="contact-us" class="contact-us section">
+    <div class="container">
+        <div class="contact-head wow fadeInUp" data-wow-delay=".4s"
+            style="visibility: visible; animation-delay: 0.4s; animation-name: fadeInUp;">
+            <div class="row">
+                <div class="col-lg-5 col-12">
+                    <div class="single-head">
+                        <div class="contant-inner-title">
+                            <h2>Biz bilan bog'lanish</h2>
+                            <p>Biz bilan quyidagi manzil va telefon raqamlari orqali bog'lanishingiz mumkin.</p>
+                        </div>
+                        <div class="single-info">
+                            <h3>Ochilish vaqtlari</h3>
+                            <ul>
+                                <li>Har kuni: 9.00 – 21.00</li>
+                                <li>Yakshanba va bayram kunlari yopiq bo'ladi</li>
+                            </ul>
+                        </div>
+                        <div class="single-info">
+                            <h3>Bog'lanish uchun</h3>
+                            <ul>
+                                <li>Oʻzbekiston, Toshkent</li>
+                                <li>Buyuk Ipak Yoʻli koʻchasi, 256</li>
+                                <li><a href="mailto:info@yourwebsite.com">manzil@email.com</a></li>
+                                <li><a href="tel:(617) 495-9400-326">(998) 99 777 00 33</a></li>
+                            </ul>
+                        </div>
+                        <div class="single-info contact-social">
+                            <h3>Ijtimoiy tarmoqlar</h3>
+                            <ul>
+                                <li><a href="#"><i class="lni lni-telegram-original"></i></a></li>
+                                <li><a href="#"><i class="lni lni-facebook-filled"></i></a></li>
+                                <li><a href="#"><i class="lni lni-instagram-filled"></i></a></li>
+                                <li><a href="#"><i class="lni lni-youtube"></i></a></li>
```

Comment:
Shu yerdagi linklarni to'ldirib qo'yish uchun Issue ochib qo'ying. Yo'qsa esdan ciqib ketadi.